app/frontend/helpers/getConversionRates.js

```diff
@@ -0,0 +1,10 @@
+const request = require('../wallet/helpers/request')
+
+const getConversionRates = async function getConversionRates() {
```

Comment:
I would add caching so each refresh of balance would not trigger an API fetch. I would create a private fetchConversionRates() method that performs the actual fetch and expose getConversionRates() which would then cache the result for let's say 5 minutes. You can inspire by how getAddressInfo and fetchAddressInfo are made in blockchain-explorer.js